galsim/gsobject.py

```diff
@@ -1238,10 +1239,17 @@ def drawImage(self, image=None, nx=None, ny=None, bounds=None, scale=None, wcs=N
         @param surface_ops  A list of operators that can modify the photon array that will be
                             applied in order before accumulating the photons on the sensor.
                             [default: ()]
+        @param sensor       An optional Sensor instance, which will be used to accumulate the
```

Comment:
You're correct about the mechanics of using the sensor with FFT methods.  I think they are accurate in the limit of the intermediate image having infinitely small pixels, regardless of `no_pixel` or not.  

But your comment reminded me that right now the intermediate image has the same size pixels as the final image.  (Which means it's not quite correct for either 'no_pixel' or 'fft' currently.)  I'd meant to add a parameter that would let the user specify how much smaller the intermediate image pixels should be.  Probably with a default of 2 or 3 maybe.

And I'll add a comment about all of that in the doc string.